frontend/packages/kubevirt-plugin/src/components/modals/scheduling-modals/shared/hooks.ts

```diff
@@ -20,7 +20,12 @@ export const useNodeQualifier = <T extends IDLabel = IDLabel>(
       const newNodes = [];
       loadedNodes.forEach((node) => {
         const nodeLabels = getLabels(node);
-        if (nodeLabels && filteredLabels.every(({ key, value }) => nodeLabels[key] === value)) {
+        if (
+          nodeLabels &&
+          filteredLabels.every(({ key, value }) =>
+            value ? nodeLabels[key] === value : !!nodeLabels[key],
```

Comment:
can the node label exist but be empty as well? Would `_.has()` make more sense here?